src/static/lib/infusion/src/framework/preferences/html/PrefsEditorTemplate-speak.html

```diff
@@ -3,9 +3,9 @@
 <div class="fl-prefsEditor-onoff">
     <input type="checkbox" id="speak-choice" class="flc-prefsEditor-speak" />
     <label for="speak-choice">
+    	<span class="flc-prefsEditor-speak-descr"></span>
         <span class="fl-prefsEditor-switch" data-checkboxStateOn="ON" data-checkboxStateOff="OFF">
             <span class="fl-prefsEditor-switch-inner"></span>
         </span>
-        <span class="flc-prefsEditor-speak-choice-label"></span>
```

Comment:
I don't think you meant to change this file, it's actually part of the the Infusion package that is being pulled in to the documentation site. If something needs to be changed the entire infusion library should be updated.